dune/iga/nurbstrimboundary.hh

```diff
@@ -0,0 +1,139 @@
+//
+// Created by Henri on 14.03.2023.
+//
+#pragma once
+
+#include "ibraGeometry.hh"
+#include "nurbstrimutils.hh"
+
+#include <clipper2/clipper.core.h>
+#include <dune/iga/nurbspatchgeometry.h>
+
+namespace Dune::IGA {
+  
+  class Boundary {
+   public:
+    static constexpr int worldDim = 2;
+    static constexpr int gridDim = 1;
```

Comment:
grid dim is here a misleading name. You should go with `dim` or `mydim`or `dimension`

Reply:
Done